### PR TITLE
Add instructions for helm-docs generation on Windows

### PR DESCRIPTION
**Pull request by Matthew-Beckett**
## Description

This PR add a Windows command example for generating helm-docs.

## Checklist

- [ ] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)
- [x] Github actions are passing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,10 +93,18 @@ All commands to be run from the root of this repo.
...
+Linux
+
   ```console
   docker run --rm -v "$PWD:/helm-docs" jnorwood/helm-docs:v1.5.0 --template-files ./ci/README.md.gotmpl
   ```
 
+Windows (PowerShell)
+
+  ```console
+  docker run --rm -v ${PWD}:/helm-docs jnorwood/helm-docs:v1.5.0 --template-files ./ci/README.md.gotmpl
```
Comment (mshero) [inline README.md:105]: What happens with the command if ´${PWD}` contains directory with spaces in the name? Will it work as expected?

Comment (max-rocket-internet) [reply README.md:105]: @Matthew-Beckett in your last PR, you had these vars shown like this `"${pwd}.Path:/helm-docs"`? I think what you have here is the same as linux as the curly braces will work there also?

File: README.md [hunk 2]
```diff
@@ -93,10 +93,18 @@ All commands to be run from the root of this repo.
 
   To generate chart `README.md` files from the [template](ci/README.md.gotmpl):
 
+Linux
```
Comment (max-rocket-internet) [inline README.md:96]:
```suggestion
Linux/Mac:
```

File: README.md [hunk 3]
```diff
@@ -93,10 +93,18 @@ All commands to be run from the root of this repo.
...
+
   ```console
   docker run --rm -v "$PWD:/helm-docs" jnorwood/helm-docs:v1.5.0 --template-files ./ci/README.md.gotmpl
   ```
 
+Windows (PowerShell)
```
Comment (max-rocket-internet) [inline README.md:102]:
```suggestion
Windows (PowerShell):
```

File: README.md (hunk 1 above)
Comment (Matthew-Beckett) [reply README.md:105]: `PS > docker run --rm -v ${PWD}:/helm-docs jnorwood/helm-docs:v1.5.0 --template-files ./ci/README.md.gotmpl`

Works on Windows with both directory paths without spaces and those with. 

```console
time="2021-05-12T08:20:31Z" level=info msg="Generating README Documentation for chart /helm-docs/stable/weblate"
time="2021-05-12T08:20:31Z" level=info msg="Generating README Documentation for chart /helm-docs/stable/front stage"
```

@max-rocket-internet in regards to your suggestion about `${PWD}` with curly braces being compatible with both platforms, I have tested this and it seems to be the case, including directories that have spaces.

Should I update the PR to include the below command as the supported cross-platform command for helm-docs generation

```console
$ docker run --rm -v ${PWD}:/helm-docs jnorwood/helm-docs:v1.5.0 --template-files ./ci/README.md.gotmpl
```

Comment (max-rocket-internet) [reply README.md:97]: We still need the quotes for Mac/Linux though. But surely with quotes it works in Powershell also?

> Should I update the PR to include the below command

If we can keep the quotes and just add the curly braces then yes

Comment (Matthew-Beckett) [reply README.md]: Done :)

**Review by max-rocket-internet — APPROVED** [review]
Nice

**Pull request merged**